anitya/templates/logs.html

```diff
@@ -94,11 +94,11 @@
 
 </div>
 
-
-<table>
+<br>
```

Comment:
Will it be appropriate to add margin instead of **br** to separate the above form the table ?